ext/intl/php_intl.stub.php

```diff
@@ -206,7 +206,7 @@ function intlcal_set(IntlCalendar $calendar, int $year, int $month, int $dayOfMo
 /** @param int|bool $value */
 function intlcal_roll(IntlCalendar $calendar, int $field, $value): bool {}
 
-function intlcal_clear(IntlCalendar $calendar, ?int $field = null): bool {}
+function intlcal_clear(IntlCalendar $calendar, ?int $field = null): true {}
```

Comment:
As far as I can see, the following functions also return only ``true``:
- ``intlcal_set()``
- ``intlcal_set_minimal_days_in_first_week()``

Reply:
Thank you! I amended my first commit to include these two functions/methods as well!